airlock_processor/tests/test_status_change_queue_trigger.py

```diff
@@ -62,6 +66,49 @@ def test_wrong_type_raises_when_getting_storage_account_properties(self):
         self.assertRaises(Exception, get_source_dest_for_copy, "accepted", "somethingelse")
 
 
+class TestFileEnumeration(unittest.TestCase):
```

Comment:
Consider adding another test for asserting the actual returned files array when mocking blob storage client

Reply:
the method is only extracting properties from the blobs that 
`container_client.list_blobs()` returns. If I mock this method, then I'm only testing my mock - no?

Reply:
There's no logic to test, so you are correct.